configs/default/JHEF-JHEH743_AIO.config

```diff
@@ -1,5 +1,12 @@
 # Betaflight / STM32H743 (SH47) 4.3.0 Sep  2 2021 / 06:34:37 (c23dd47f4) MSP API: 1.44
 
+#define USE_ACC_SPI_MPU6000
+#define USE_GYRO_SPI_MPU6000
+#define USE_BARO_BMP280
+#define USE_MAX7456
+#define USE_FLASH_M25P16
+#define USE_LED
```

Comment:
Please remove `USE_LED` as it's a user setting in the cloud build.

https://github.com/betaflight/betaflight/blob/master/docs/Manufacturer%20Design%20Guidelines.md#42-definitions-for-unified-targets

Reply:
Fixed